bindings/bind_py_structure_manager.cc

```diff
@@ -642,7 +646,7 @@ namespace rascal {
         "__getitem__",
         [](ManagerCollection_t & v, int index) { return v[index]; },
         py::keep_alive<0, 1>());
-
+    manager_collection.def("__len__", &ManagerCollection_t::size);
```

Comment:
hmm, I'm confused as to what this "len" actually corresponds to -- is it the number of centres or the number of structures in the collection?

Reply:
added documentation